client/src/common/components/Header.tsx

```diff
@@ -1,9 +1,162 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { Link } from 'react-router-dom';
+// import { FaTimes } from 'react-icons/fa';
+// import { FaBars } from 'react-icons/fa';
 
 const Header = () => {
-	// test tailwind
-	return <div className="bg-header_color text-white">This is Header</div>;
+	const [isClosed, setClosed] = useState(true);
 
+	useEffect(() => {
+		document.addEventListener('mousedown', () => {
+			setClosed(true);
+		});
+	});
+
+	const mymenu = React.useRef() as React.MutableRefObject<HTMLDivElement>;
+
+	return (
+		<div
+			className="bg-header_color
+						text-white flex
+						justify-between
+						items=center 
+						font-black 
+						w-screen
+						h-14
+						md:h-16
+						p-4
+						mx-auto
+						"
+		>
+			<div
+				className="flex-grow 
+							text-lg
+							w-screen
+							md:text-3xl
+							"
+			>
+				<Link to="/">Senpai-Kohai</Link>
+			</div>
+			{/* {!hasValidToken ? ( */}
+			{/* <div className="text-base 
+								box-border 
+								border-2 
+								w-20 
+								rounded-full 
+								cursor-pointer 
+								font-base 
+								hover:bg-primary_bg_color 
+								mr-5 
+								text-center">
+					<Link to="/login">Login</Link>
+				</div>
+				<div className="bg-primary_bg_color text-header_color w-20 text-base box-border border-2 rounded-full cursor-pointer hover:bg-secondary_bg_color text-center">
+					<Link to="/signup">Sign up</Link>
+				</div> */}
+			{/* ) : ( */}
+
+			<div
+				className="text-xs
+							box-border 
+							border-2 
+							md:w-40
+							p-0.5
+							md:p-0
+							w-40
+							rounded-full 
+							cursor-pointer 
+							hover:bg-primary_bg_color 
+							mr-10 
+							text-center
+							md:text-base
+							ml-6
+							"
+			>
+				Sign Out
```

Comment:
```suggestion
				Log out
```
Let's follow the design on our Figma